probes/fuzzedWithClusterFuzzLite/impl.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package fuzzedWithClusterFuzzLite
+
+import (
+	"embed"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/utils"
+)
+
+//go:embed *.yml
+var fs embed.FS
+var probe = "fuzzedWithClusterFuzzLite"
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	//nolint:wrapcheck
+	return utils.FuzzerRun(raw, fs, probe, "ClusterFuzzLite")
```

Comment:
so right now the fuzzer names live here:
https://github.com/ossf/scorecard/blob/71e8d5e3c38467bbb590445276337f383397b2b3/checks/raw/fuzzing.go#L30-L39

Is it worth putting them in some `internal` module, so we can de-duplicate the string itself and just refer to it as a const?

Reply:
yeah I would like to refer to them instead of re-hardcoding them. Shall I try that in this PR or another one?

Reply:
I'm fine with this PR

Reply:
They can't be internal I think. If they are, they won't be accessible from the probes package. I can make them public to avoid duplication. Wdut?

Reply:
They can be internal, just internal at a higher level (e.g. top level `internal`)

Reply:
if we put the probes in there, sure. That seems out of scope of this PR. What organization would you like?

Reply:
Created https://github.com/ossf/scorecard/issues/3407 for tracking.